consensus/src/liveness/round_state.rs

```diff
@@ -254,9 +259,12 @@ impl RoundState {
     /// Notify the RoundState about the potentially new QC, TC, and highest committed round.
     /// Note that some of these values might not be available by the caller.
     pub fn process_certificates(&mut self, sync_info: SyncInfo) -> Option<NewRoundEvent> {
+        // Question: We are comparing ordered round with committed round. Is this accurate?
```

Comment:
I think it is a naming issue. We should rename `highest_committed_round` to `highest_ordered_round` ? cc @zekun000

Reply:
Changed the if statement to compare with highest_committed_round instead of highest_ordered_round.